src/modules/ProductLicense.rb

```diff
@@ -1145,12 +1145,13 @@ def AskLicenseAgreement(src_id, dir, patterns, action, enable_back, base_product
 
       licenses_ref = arg_ref(licenses)
 
-      label = Pkg::SourceGeneralData(src_id)["name"]
       title = _("License Agreement")
 
-      if !label.empty?
-        # %s is an extension name, e.g. "SUSE Linux Enterprise Software Development Kit"
-        title = _("%s License Agreement") % label
+      if src_id
+        label = Pkg::SourceGeneralData(src_id)["name"]
+        # TRANSLATORS: %s is an extension name
+        # e.g. "SUSE Linux Enterprise Software Development Kit"
```

Comment:
Are there any limitations for length? As short as possible? Should be in the comment